coordinator/src/main/java/org/apache/uniffle/coordinator/SelectStorageStrategy.java

```diff
@@ -17,24 +17,12 @@
 
 package org.apache.uniffle.coordinator;
 
+import java.util.List;
 import java.util.Map;
 
-import org.apache.uniffle.common.RemoteStorageInfo;
 import org.apache.uniffle.coordinator.LowestIOSampleCostSelectStorageStrategy.RankValue;
 
 public interface SelectStorageStrategy {
 
-  RemoteStorageInfo pickRemoteStorage(String appId);
-
-  void incRemoteStorageCounter(String remoteStoragePath);
-
-  void decRemoteStorageCounter(String storagePath);
-
-  void removePathFromCounter(String storagePath);
-
-  Map<String, RemoteStorageInfo> getAvailableRemoteStorageInfo();
-
-  Map<String, RemoteStorageInfo> getAppIdToRemoteStorageInfo();
-
-  Map<String, RankValue> getRemoteStoragePathRankValue();
+  List<Map.Entry<String, RankValue>> readAndWrite(String path);
```

Comment:
I have a little doubt, why do the `readAndWrite` become the only interface method for SelectStorageStrategy? I can rank by many methods, it's not necessary to read and write.

Reply:
The meaning of the `readAndWrite` interface is to perform anomaly detection of remote paths. For hdfs, reading and writing files is a more direct way to detect, so I took this name, but the real comparison method comes from `sortPathByRankValue`, which I do not have. The reason for defining it as an interface is because the objects to be sorted may be different, and this is left to different storage methods for implementation.

Reply:
`readAndWrite` is not a good name. It's strange for the interface `SelectStorageStrategy`.

Reply:
OK, updated.

Reply:
Should we add a method `pickStorage` or `selectStorage` for this interface?

Reply:
Yes, that might make more sense for this interface, I will added.